pymeasure/instruments/mksinst/mks937b.py

```diff
@@ -0,0 +1,189 @@
+#
+# This file is part of the PyMeasure package.
+#
+# Copyright (c) 2013-2022 PyMeasure Developers
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+import re
+
+from pymeasure.instruments import Channel, Instrument
+from pymeasure.instruments.validators import strict_discrete_set
+
+
+_ion_gauge_status = {"Wait": "W",
+                     "Off": "O",
+                     "Protect": "P",
+                     "Degas": "D",
+                     "Control": "C",
+                     "Rear panel Ctrl off": "R",
+                     "HC filament fault": "H",
+                     "No gauge": "N",
+                     "Good": "G",
+                     "NOT_IONGAUGE": "NAK152",
+                     "INVALID COMMAND": "NAK160",
```

Comment:
The "INVALID COMMAND" line could be dropped now. This should never trigger.

Reply:
I'd keep it in for completeness.

Reply:
Agree, this doesn't hurt to keep.